obspy/scripts/runtests.py

```diff
@@ -3,774 +3,176 @@
 """
 A command-line program that runs all ObsPy tests.
 
-All tests in ObsPy are located in the tests directory of the each specific
-module. The __init__.py of the tests directory itself as well as every test
-file located in the tests directory has a function called suite, which is
-executed using this script. Running the script with the verbose keyword exposes
-the names of all available test cases.
+All tests in ObsPy are located in the tests directory of each specific
+module.
 
-:copyright:
-    The ObsPy Development Team (devs@obspy.org)
-:license:
-    GNU Lesser General Public License, Version 3
-    (https://www.gnu.org/copyleft/lesser.html)
+A few important command line arguments:
+    --help : Print help message an exit.
+    --no-report, --report : Do not ask and automatically (not) submit report
+            to ObsPys' report server.
+    --cov : Calculate and display test coverage.
+    --network : Only run network tests (requires an internet connection).
+    --all : Run both network and non-network tests.
+    --keep-images : Store all images generated by tests in obspy's directory.
+
+Also note, the test runner uses pytest under the hood so any pytest command
+line argument is also accepted.
 
 .. rubric:: Examples
 
 (1) Run all local tests (ignoring tests requiring a network connection) on
     command line::
-
         $ obspy-runtests
-
-    or via Python interpreter
-
-    >>> import obspy.core
-    >>> obspy.core.run_tests()  # DOCTEST: +SKIP
-
-(2) Run all tests on command line::
-
+(2) Run all tests on command line (including network tests)::
         $ obspy-runtests --all
-
-    or via Python interpreter
-
-    >>> import obspy.core
-    >>> obspy.core.run_tests(all=True)  # DOCTEST: +SKIP
-
-(3) Verbose output::
-
-        $ obspy-runtests -v
-
-    or
-
-    >>> import obspy.core
-    >>> obspy.core.run_tests(verbosity=2)  # DOCTEST: +SKIP
-
-(4) Run tests of module :mod:`obspy.io.mseed`::
-
-        $ obspy-runtests obspy.io.mseed.tests.suite
-
-    or as shortcut::
-
-        $ obspy-runtests io.mseed
-
-(5) Run tests of multiple modules, e.g. :mod:`obspy.io.wav` and
+(3) Run tests of module :mod:`obspy.io.mseed`::
+        $ obspy-runtests io/mseed
+(4) Run tests of multiple modules, e.g. :mod:`obspy.io.wav` and
     :mod:`obspy.io.sac`::
+        $ obspy-runtests io/wav obspy/io/sac
+(5) Run a specific test case::
+        $ obspy-runtests core/tests/test_stats.py::TestStats::test_init
+(6) Run tests and print a coverage report to screen and save coverage.xml
+    with pytest-cov plugin:
+        $ obspy-runtests --coverage
```

Comment:
--cov or --coverage?

Reply:
Ya, I guess we aren't really supporting running the tests from python anymore; just pytest or obspy-runtests from the command line.